dm/tests/full_mode/data/db1.prepare.user.sql

```diff
@@ -2,6 +2,5 @@ drop user if exists 'dm_full';
 flush privileges;
 create user 'dm_full'@'%' identified by '123456';
 grant all privileges on *.* to 'dm_full'@'%';
-revoke replication slave, replication client on *.* from 'dm_full'@'%';
```

Comment:
Now I think if we can dump without those privileges before, we should not raise an warning or error now